sdk/include/game-contract-sdk/dispatcher.hpp

```diff
@@ -38,6 +38,46 @@ bool execute_action(eosio::name self, eosio::name code, void (game::*func)(Args.
 
 } // namespace game_sdk
 
+#ifdef IS_DEBUG
```

Comment:
hm, I think, approach with creating macro like EXTRA_ACTIONS with debug actions under IS_DEBUG and empty else is better here, that allow to add only debug actions without full dispatcher copy.

```c++
#ifdef IS_DUBUG
#define EXTRA_ACTIONS \
            case "pushnrandom"_n.value:                                                                                \
                game_sdk::execute_action<TYPE>(eosio::name(receiver), eosio::name(code), &TYPE::push_next_random);     \
                break;                                                                                                 \
            case "pushprng"_n.value:                                                                                   \
                game_sdk::execute_action<TYPE>(eosio::name(receiver), eosio::name(code), &TYPE::push_to_prng);         \
                break; 
#else
#define EXTRA_ACTIONS
#endif
```
and

```c++
....
case "close"_n.value:                                                                                      \
                game_sdk::execute_action<TYPE>(eosio::name(receiver), eosio::name(code), &TYPE::close);                \
                break;      

EXTRA_ACTIONS

default:
eosio::eosio_exit(1);
```